framework/assets/yii.activeForm.js

```diff
@@ -195,6 +198,18 @@
                     });
                     $form.on('submit.yiiActiveForm', methods.submitForm);
                 }
+                if (settings.scrollToError) {
+                    $form.on('afterValidate.yiiActiveForm', function(event, messages, errorAttributes) {
```

Comment:
I'd like to see this fixed in 2.0.5. I can put it back to where it was, if this is a problem.